compiler/testData/codegen/bytecodeText/jvm8/jvmDefault/compatibility/simpleFunctionWithAbstractOverride.kt

```diff
@@ -18,7 +17,7 @@ interface KInterface2 : KInterface  {
 // +
 // 0 INVOKESTATIC KInterface2.access\$test2\$jd
 // =
-// 1 INVOKESTATIC
+// 1 INVOKESTATIC KInterface
```

Comment:
The test was over-specified: the IR backend produces a nullcheck on `this` in `KInterface$DefaultImpls$test2` which is a static method.